pkg/api/v1/atlasproject_types.go

```diff
@@ -65,6 +65,16 @@ type AtlasProjectSpec struct {
 	// CloudProviderAccessRoles is a list of Cloud Provider Access Roles configured for the current Project.
 	CloudProviderAccessRoles []CloudProviderAccessRole `json:"cloudProviderAccessRoles,omitempty"`
 
+	// AlertConfiguration is a list of Alert Configurations configured for the current Project.
+	AlertConfigurations []AlertConfiguration `json:"alertConfigurations,omitempty"`
+
+	// AlertConfigurationSyncEnabled is a flag that enables/disables Alert Configurations sync for the current Project.
+	// If true - project alert configurations will be synced according to AlertConfigurations.
+	// If not - alert configurations will not be modified by the operator. They can be managed through API, cli, UI.
+	//kubebuilder:default:=false
+	// +optional
+	AlertConfigurationSyncEnabled bool `json:"alertConfigurationEnabled,omitempty"`
```

Comment:
Won't you also update the name for the CRD?

Reply:
Yeah 😄 
Done